documents/DeveloperGuide/MainPage.md

```diff
@@ -2,25 +2,213 @@
 
 MaterialX is an open standard for representing rich material and look-development content in computer graphics, enabling its platform-independent description and exchange across applications and renderers.  Launched at [Industrial Light & Magic](https://www.ilm.com/) in 2012, MaterialX has been a key technology in their feature films and real-time experiences since _Star Wars: The Force Awakens_ and _Millennium Falcon: Smugglers Run_.  The project was released as open source in 2017, with companies including Sony Pictures Imageworks, Pixar, Autodesk, Adobe, and SideFX contributing to its ongoing development.  In 2021, MaterialX became the seventh hosted project of the [Academy Software Foundation](https://www.aswf.io/).
 
+# Setup and Build MaterialX
```

Comment:
To keep the flow of the documentation as clear as possible, I would recommend omitting this new section heading, leaving `Quick Start for Developers` as the first section after the `Overview`.